design-reviews/recipes/technical-spike.md

```diff
@@ -0,0 +1,33 @@
+# Technical Spike
+
+From [Wikipedia](<https://en.wikipedia.org/wiki/Spike_(software_development)>)...
+
+A spike in a sprint can be used in a number of ways:
+
+- As a way to familiarize the team with new hardware or software
+- To analyze a problem thoroughly and assist in properly dividing work among separate team members.
+- Spikes tests can also be used to mitigate future risk, and may uncover additional issues that have escaped notice.
+
+A distinction can be made between technical spikes and functional spikes. The technical spike is used more often for evaluating the impact new technology has on the current implementation. A functional spike is used to determine the interaction with a new feature or implementation.
+
+## Deliverable
+
+Generally the deliverable from a Technical Spike should be a document detailing what was evaluated and the outcome of that evaluation. The specifics contained in the document will vary, but there are some general principals that might be helpful.
+
+In addition, the "technical-spike-samples" folder contains sample documents from previous spikes. "istio-spike.docx" is the most recent. Some information has been redacted.
```

Comment:
Please don't add binary files to the repo as they can't be git-diffed... and we also can't PR review them without downloading the branch, nor can we view them in the repo.

If you want to add examples, please convert them to proper markdown files.  
In general we should encourage the use of markdown for documentation that makes it into repos.

Reply:
Samples have been removed and a note that they are available on request.